lms/djangoapps/teams/views.py

```diff
@@ -59,6 +62,25 @@
 MAXIMUM_SEARCH_SIZE = 100000
 
 
+@receiver(pre_save, sender=CourseTeam)
+def team_pre_save_callback(sender, instance, **kwargs):  # pylint: disable=unused-argument
```

Comment:
I thought it made more sense to use pre-save callbacks at first, but you're right in that it probably actually makes more sense to use post-save callbacks for tracking these fields. I've updated the code to use the post-save signals instead.